### DISCUSSION TITLE
Set a default country for the store for tax?

### DISCUSSION THREAD

**Discussion by johnrisby** [swellstores/community#369] [Help] [answered]
Hi all,

Is it possible to have a default store country so it applies taxes without having a create/login a user?

Thanks

**Comment by johnrisby** [comment, marked as answer]
Actually, I've done it with this, although I need to add a check of course to make sure there's not already a shipping address set (change to billing address if your tax is based on that).

```
  useEffect(() => {
    (async () => {
      await swell.cart.update({
        shipping: {
          country: "GB"
        }
      });
    })();
  }, []);
```
  
  Is there any potential problems with this approach (once I add the check for an existing country)?
  
  Thanks

**Reply by logeshswell** [reply]
There shouldn't be any issue with this approach to my knowledge.
